gc/base/Configuration.cpp

```diff
@@ -511,16 +511,33 @@ MM_Configuration::createParallelDispatcher(MM_EnvironmentBase *env, omrsig_handl
 
 #if defined(J9VM_OPT_CRIU_SUPPORT)
 void
-MM_Configuration::adjustGCThreadCountOnCheckpoint(MM_EnvironmentBase* env)
+MM_Configuration::adjustGCThreadCountForCheckpoint(MM_EnvironmentBase* env)
 {
 	MM_GCExtensionsBase* extensions = env->getExtensions();
+	MM_ParallelDispatcher* dispatcher = extensions->dispatcher;
 
-	extensions->dispatcher->contractThreadPool(env, extensions->checkpointGCthreadCount);
+	dispatcher->contractThreadPool(env, extensions->checkpointGCthreadCount);
+
+	/* This line is temporary, it's required to prevent thread pool growth beyond the initial pool sizing done at startup.
+	 * It's required until dependent functional components (beyond just Dispatcher) are ensured to be ok with the growth.
+	 */
```

Comment:
- exceeds more than 100 chars, too long.
- `ok` should be replaced with a better (technical) phrase.

Reply:
👍